packages/web/src/common/store/pages/feed/lineup/sagas.ts

```diff
@@ -46,9 +46,8 @@ function* getTracks({
   const apiClient = yield* getContext('apiClient')
   const filter = filterMap[filterEnum]
 
-  // NOTE: The `/feed` does not paginate, so the feed is requested from 0 to N
   const params: GetSocialFeedArgs = {
-    offset: 0,
+    offset,
```

Comment:
yooo it just works now?

Reply:
yes!!

Reply:
how much faster is it?

Reply:
on mobile i notice that tracks load much smoother, especially deep into the feed, try it out on staging in a sec here :)